tests/jsmarionette/client/sockit-to-me/package.json

```diff
@@ -6,7 +6,7 @@
   "main": "build/Release/sockit",
 
   "devDependencies": {
-    "node-gyp": "~0.10.6"
+    "node-gyp": "3.0.1"
```

Comment:
@nullaus My only contention is that since sockit-to-me really only changes whenever there is a new Node.js version, that its version stay in line with Node's. That way if someone can't upgrade to Node.js 4.0.0, they can use the 0.x branch of sockit-to-me, and if someone can't use Node.js 5.0.0, they can use the 4.x branch of sockit-to-me. Does that sound reasonable?

I don't have any opinions about engines really, more just that sockit-to-me should be more semver reliable.